pkg/cloud/graph.go

```diff
@@ -49,7 +49,7 @@ func (c *AzureClient) CreateApplication(ctx context.Context, displayName string)
 	}
 	appPostOptions.Body.SetDisplayName(to.StringPtr(displayName))
 
-	log.Debugf("Creating application with display name=%s", displayName)
+	mlog.Debug("Creating application", "display name", displayName)
```

Comment:
As we move to structured logging, should the key be camel case? `displayName` instead of `display name`?

Reply:
I am happy either way: which way do you prefer?

Reply:
Let's use the camel case keys